file_sharing.cpp

```diff
@@ -334,13 +342,171 @@ static optional<FileSharing::OnlineModInfo> parseModInfo(const vector<string>& f
   return none;
 }
 
+static string shortDescription(string text, int max_lines = 3) {
```

Comment:
Maybe rename the function to what it actually does, which is returning the first max_lines lines from text?

Reply:
Sure